### ISSUE TITLE
ATO's set to Nothing do not operate

### ISSUE THREAD

**Issue by mginster** [reef-pi/reef-pi#2119]
I wouldn't say this is a bug, maybe a feature request. 

If you set an ATO to Control:Nothing, it does nothing (never graphs the sensor or alerts).  The reason I would want to do this is for a sump high/low sensor that I just want an alert on. 

Example:
This is my SUMP LOW ATO and has been activated for a few minutes in a sensor triggered state (sump is indeed low).  No graph/no alerts.
![image](https://github.com/reef-pi/reef-pi/assets/20690603/d721cdd5-ae32-4e60-ad05-56dff620a51c)

Example with equipment in Control:
As expected, it works fine, graphs/alerts etc when you actually have it set to Equipment.
![image](https://github.com/reef-pi/reef-pi/assets/20690603/547a5835-a872-4737-87b6-037135ed14b0)

This is easy to workaround just telling it to turn on a non-existent dummy plug, but would be cleaner to allow alerts to operate with Nothing set.  I know you can also set it to a Macro and alert there and then disable the ATO....however my other Issue submitted shows Macros always trigger for some reason.

**Comment by ranjib** [comment]
Can you try with dummy /non existent equipment instead of nothing

**Comment by mginster** [comment]
Yes, that does work. It’s an easy workaround. Just suggesting this change to make it cleaner and easier for others doing similar. Up to you to include or not.
